src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+
+    public static Colors getRandomColor() {
+        int index = new Random().nextInt(Colors.values().length);
```

Comment:
Consider using a single instance of `Random` instead of creating a new one each time. This can be done by making `Random` a class-level variable.